vocabs/isoblue/index.js

```diff
@@ -0,0 +1,74 @@
+var libvocab = require('../../lib/vocab')('oada');
+
+// Only importing the essentials, more complext definitions may need more imports
+// similar to the main oada index.js this is based off of
+const {register,  override} = libvocab;
+
+// Custom location schema
+register('lat', override('latitude', {
+    description: 'Short form of latitude term, otherwise the same. ' +
+                 'Used in ISOBlue CAN data.'
+}));
+  
+register('lng', override('longitude', {
+    description: 'Short form of longitude term, otherwise the same. ' +
+                 'Used in ISOBlue CAN data.'
+}));
+
+
+// Custom CAN schema
+register('is_error_frame', override('generic-data-value', {
+    description: 'Error frame flag for can frame',
+    type: Boolean,
+}));
+
+register('is_extended_frame', override('generic-data-value', {
+    description: 'Extended fram flag for can frame',
+    type: Boolean,
+}));
+
+register('arbitration_id', override('generic-data-value', {
+    description: 'CAN ID portion of extended can frame',
+    type: Number,
+}));
+
+register('dlc', override('generic-data-value', {
+    description: 'Data Length Code portion of extended can frame',
+    type: Number,
+}));
+
+register('payload', override('generic-data-value', {
+    description: 'String representaiton can frame payload in hexadecimal',
+    type: String,
+}));
+
+register('is_remote_frame', override('generic-data-value', {
+    description: 'Remote frame flag from can frame',
+    type: Boolean,
+}));
+
+// Custom heartbeat schema
+register('backlog', override('generic-data-value', {
+    description: "Length of backlog in computer systems. Used in Isoblue"+
+                 " to send amount of unsent messages",
+    type: Number,
```

Comment:
Same issue here as the issue in the oada vocab below: a generic-data-value isn't a number, it's an object with "value" and "units".  In this case, the units would be "count".